sdk/src/types/api/core.rs

```diff
@@ -339,31 +339,33 @@ pub struct SubmitBlockResponse {
 #[derive(Clone, Copy, Debug, Eq, PartialEq, Serialize, Deserialize)]
 #[serde(rename_all = "camelCase")]
 pub enum BlockState {
-    // Stored but not confirmed.
+    /// Stored but not confirmed.
     Pending,
-    // Confirmed with the first level of knowledge.
+    /// TODO
```

Comment:
TODO

Reply:
This variant is not yet in the TIP 🤷🏻

Reply:
probably should find a good description for it before merging tho

Reply:
Done